model-optimizer/extensions/back/MaxPool.py

```diff
@@ -25,3 +26,14 @@ def replace_pattern(self, graph: Graph, match: dict):
         del node['pool_method']
         if 'exclude_pad' in node:
             del node['exclude_pad']
+
+        # adding missed outputs for MaxPool node
+        if node.out_port(0).disconnected():
+            output = Result(node.graph, {'name': node.name + '/Result_port_0/',
+                                         'keep_output_port': node.has_and_set('remove_values_output')}).create_node()
+            node.out_port(0).get_connection().set_destination(output.in_port(0))
+
+        if node.out_port(1).disconnected():
+            output = Result(node.graph, {'name': node.name + '/Result_port_1/',
```

Comment:
nit: Why do we need such blank `Result` only for `MaxPool` but not for `AvgPool`?

Reply:
`AvgPool-1` does not have optional outputs.

Reply:
@yekruglov, please clarify why we do this? I see the same stuff for `TopK`.
I have not found that this attribute is used somewhere.